superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -29,10 +29,7 @@ import { uniq } from 'lodash';
 
 const propTypes = {
   name: PropTypes.string,
-  operatorType: PropTypes.string,
-  columnType: PropTypes.string,
   sliceId: PropTypes.number,
-  width: PropTypes.number,
   subscriptionList: PropTypes.arrayOf(PropTypes.object),
   value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   overrides: PropTypes.object,
```

Comment:
What is the use or overrides and value ?

Reply:
removed